Library/Formula/norm.rb

```diff
@@ -1,8 +1,9 @@
 class Norm < Formula
   desc "NACK-Oriented Reliable Multicast"
   homepage "https://www.nrl.navy.mil/itd/ncs/products/norm"
-  url "https://downloads.pf.itd.nrl.navy.mil/norm/archive/src-norm-1.5b3.tgz"
-  sha256 "ec2014f57be12ea3ade0685faa1e6e161f168311c8975093bce33a6e1dc5e77c"
+  url "https://downloads.pf.itd.nrl.navy.mil/norm/archive/src-norm-1.5r5.tgz"
+  version "1.5r5"
```

Comment:
@bebopagogo I've created a pull request #41554 - I can't speak for the homebrew team, but I guess the best way to do it is by submitting a pull request like I did.
I'm happy to do it, but you can of course also do it yourself; https://git.io/brew-docs